.golangci.yml

```diff
@@ -15,7 +15,7 @@ linters:
   enable:
     - misspell
     - gofmt
-    - deadcode
+    - unused
```

Comment:
Upgrading golangci-lint to 1.51.0, and use `unused` instead as `deadcode` is deprecated since v1.49.0.